app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -7,8 +7,6 @@ module ActionTriggering
         after_create :plan_hook_action
         after_update :plan_hook_action
         after_destroy :plan_hook_action
```

Comment:
Will this block get evaluated when the module gets prepended instead of getting included? If not, this needs to be converted to something like 
```ruby
def self.prepended(base)
  base.after_create :plan_hook_action
  base.after_update :plan_hook_action
  base.after_destroy :plan_hook_action
end
```